src/components/InstallerView.jsx

```diff
@@ -0,0 +1,31 @@
+import viewStyles from '../styles/view'
+import React, { Component } from 'react'
+import { translate } from 'cozy-ui/react/helpers/i18n'
+import classNames from 'classnames'
+
+class InstallerView extends Component {
+  doInstall () {
+    // get the values from the fields
+    const slug = document.querySelector('#slug').value
```

Comment:
See https://facebook.github.io/react/docs/forms.html#controlled-components

As React/preact uses a specific paradigm to manage states in component, we recommend to use the same paradigm for the special html elements that manage state.